### PR TITLE
[JA] Add `Client/Keyboard_shortcuts`

### PR DESCRIPTION
**Pull request by yuzupon1133**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

😓

### CODE REVIEW

**Comment by yuzupon1133** [comment]
image
where wrong...

**Comment by TicClick** [comment]
it's in the "Files changed" tab: 
image

**Comment by yuzupon1133** [comment]
thanks

File: wiki/Client/Keyboard_shortcuts/ja.md [hunk 1]
```diff
@@ -0,0 +1,293 @@
...
+| `Ctrl` + `,` | 選択範囲を反時計回りに90度回転させます。 |
+| `Ctrl` + `.` | 選択範囲を時計回りに90度回転させます。 |
+| `Ctrl` + `H` | 選択範囲を水平方向に反転します。 |
+| `Ctrl` + `J` | 選択範囲を上下に反転します。 |
+| `Ctrl` + `G` | 選択範囲のオブジェクトの順番を逆にします。つまり、最初のオブジェクトは最後のオブジェクトに移動するというようにします。 |
+| `Ctrl` + `Shift` + `R` | 選択したヒットオブジェクトを指定した角度だけ回転させます（ダイアログが開きます）。 |
+| `Ctrl` + `Shift` + `S` | 選択したヒットオブジェクトを指定した量だけ拡大縮小します（ダイアログが開きます）。 |
+| `Ctrl` + `Shift` + `D` | 指定した数のポイントを持つサークルの図形を生成します（ダイアログが開きます）。 |
+| `Ctrl` + `Shift` + `F` | 選択したスライダーをサークルのストリームに変換します（ダイアログが開きます）。 |
+| `Ctrl` + `左/右/上/下矢印` | 選択したヒットオブジェクトをその方向に移動します。 |
```
Comment (peppy) [inline wiki/Client/Keyboard_shortcuts/ja.md:248]:
```suggestion
| `Ctrl` + `左/右/上/下矢印` | 選択したヒットオブジェクトをその方向に移動します。 (osu!, osu!taiko, osu!catch)|
```

File: wiki/Client/Keyboard_shortcuts/ja.md [hunk 2]
```diff
@@ -0,0 +1,293 @@
...
+| `Ctrl` + `C` | 選択範囲をコピー |
+| `Ctrl` + `V` | 現在の場所にスプライトを貼り付けます（貼り付けたスプライトは、ビートマップが保存されて再読み込みされるまで、レイヤーリストに表示されません）。 |
+| `Delete` | 選択したスプライトをレイヤーから削除します。 |
+| `W`, `A`, `S`, `D` | 選択したスプライトを1[osu!pixel](/wiki/osupixel)ずつ移動します。 |
+
+### タイミング
+
+| キー | 動作 |
+| :-- | :-- |
+| `T` | ビートからBPMを設定します。 |
```
Comment (peppy) [inline wiki/Client/Keyboard_shortcuts/ja.md:289]:
```suggestion
| `T` | ビートからBPMを設定します。 |
```

It also sets offset, so best mention both BPM and offset here

**Comment by yuzupon1133** [comment]
changed.

**Review by peppy — APPROVED** [review]

**Pull request merged**